### PR TITLE
Guardian Changes

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Based on feedback and my own experience, healer and shield are fucking boring to play as. The most interesting to play (in my opinion) is the Ranged/Scout because it has utility other than just murdering things. With that in mind, I am reworking the guardians as follows.

Standard - Unchanged

Chaos- Combines Fire and Bluespace. Teleports targets on hit as well as igniting mobs on touch/bump. Auto extinguishes its user every tick. Slightly lowered resistance and attack damage.

Ranged/Scout - Slightly buffed attack damage (5 a hit vs 4), slightly buffed damage resistance. Surveillance snares now alert the user as well.

Support - Can toggle between healing and combat mode (healing attacks/slow vs normal attacks/normal speed). Can mark a turf/leave a beacon and warp mobs/objects to it on shift+click (after a standing still delay twice as long as handcuffs). Hopefully kidnapping shenanigans.

Explosive - Unchanged.

So Healer got a new power/toggle to combat, Fire/Bluespace got merged, Fast and Shield got axed (for being redundant and boring, respectively).

In addition, when the guardians user is in crit and the guardian takes damage, the user will be dealt an additional 50% damage as cloneloss.

Changelings can not use guardian spirits.

There is a check to prevent a single person from having more than one Guardian.

Guardians no longer break if they recall while buckled.

The tech themed variant has been renamed from parasitic nanoswarm to Holoparasites.

Added Holoparasites to the traitor uplink for 12 telecrystals. Probably a bit low, but this was the result of an OOC poll on how much they should cost (enough TC left over for the traitor to have a bit of their own gear, but no gun)

New sprites for the tech themed guardians (not sure how to credit these, WJ made a sprite, I edited it, and my girlfriend then redid most of it).

### CODE REVIEW

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -199,63 +206,106 @@
...
+			var/mob/living/carbon/C = target
+			C.adjustBruteLoss(-5)
+			C.adjustFireLoss(-5)
+			C.adjustOxyLoss(-5)
+			C.adjustToxLoss(-5)
+
+/mob/living/simple_animal/hostile/guardian/healer/verb/ToggleMode()
+	set name = "Toggle Mode"
+	set category = "Guardian"
+	set desc = "Toggle between combat and scout modes."
```
Comment (Anonus) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: I'm pretty sure you mean healing

**Comment by Anonus** [comment]
Shield would be cool if it could move through people(but not people through it) and its attacks threw people away from it, but as is it's shit, so shit.

**Comment by KorPhaeron** [comment]
@Anonus 

I thought about doing that, but realized

A) It'd be overlapping with the Chaos one in throwing people 

B) Making it more effective still leaves it with nothing to do but murder people, and if it's too good at it's job it will just lead to unstoppable killing sprees. It just didn't seem worth the headache of trying to balance. Maybe I'll rework it eventually so it can place barriers that it can pass through like swarmers, but for now it's getting dropped.

**Comment by PKPenguin321** [comment]
>after a standing still delay twice as long as handcuffs

this will literally never be used because the guardian spirit will always be less than a screen away from their master and it takes so fucking long that the master may as well just pick up whatever they're warping and carry it there

**Comment by KorPhaeron** [comment]
@PKPenguin321 

The way I imagined it was yeah, warping the user (and friends!) to a secret base after some nefarious deed, or beating someone to crit, warping them to base, then healing them. Sometimes the thing you want to warp (a body, a prisoner, a fugitive) is something you don't want carried there normally.

I could lower the delay a bit though.

**Comment by Aloraydrel** [comment]
R e m o v e  S h i e l d

**Comment by KorPhaeron** [comment]
Ready for merge/review @RemieRichards @Miauw62 

Though maybe it would be better as a test PR first since it effects traitor uplinks @MrStonedOne

**Comment by peoplearestrange** [comment]
I have to say I love the concept for all of these and they can be pretty fun to play. However, the sprites look god awful. They kinda look like a crappy ghost. Obviously its hard to represent a swarm of nanobots but still, it looks too spirit like rather than tech like.

**Comment by KorPhaeron** [comment]
@peoplearestrange 

It IS a crappy ghost (made crappier by the colour var). I still don't have sprites for the tech theme.

**Comment by KorPhaeron** [comment]
Guess I'll try @Razharas as well since you're back now

Promise it works, it's been on the server for a couple days now which exposed the bugs very quickly.

**Comment by Anonus** [comment]
The holoparasites need to bounce.

**Comment by Razharas** [comment]
No idea what a guardian is so whatevs

**Comment by Razharas** [comment]
#yomo

**Pull request merged**